app-frontend/src/app/pages/projects/edit/aoi-parameters/aoi-parameters.controller.js

```diff
@@ -112,28 +136,18 @@ export default class AOIParametersController {
         });
     }
 
-    updateProjectAOIs(polygonCoordinates) {
+    updateProjectAOIs(multipolygon) {
         if (this.projectAois && this.projectAois.length === 1) {
             // update existing aoi
             let aoiToUpdate = this.projectAois[0];
-            if (polygonCoordinates.length) {
-                aoiToUpdate.area = {
-                    geom: {type: 'MultiPolygon', coordinates: polygonCoordinates},
-                    srid: 3857
-                };
-            } else {
-                this.$log.error('An AOI must be composed of at least one polygon');
-            }
+            aoiToUpdate.area = multipolygon;
             this.aoiService.updateAOI(aoiToUpdate).then(() => {
```

Comment:
This isn't a change you made but I think fixing this is within the scope of this PR. This call to `this.aoiService.updateAOI` causes us to persist this AOI whenever we finish editing a shape. We have that big button at the bottom of the page which saves the other params on this page which are only persisted when we click that button. We should only save the AOI when the button is clicked along with the other params.